pkg/pravega/bookie.go

```diff
@@ -157,6 +165,32 @@ func makeBookiePodSpec(clusterName string, bookkeeperSpec *v1alpha1.BookkeeperSp
 				},
 			},
 		},
+		Affinity: &corev1.Affinity{
+			PodAntiAffinity: &corev1.PodAntiAffinity{
+				PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{
+					{
+						Weight: 100,
```

Comment:
What's the criteria for choosing 100 for the weight? I'm trying to understand why `100` and not `99` or `10`or something else.

Reply:
As we only have one affinity rule, I set it to 100 (max priority) following the example in https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#an-example-of-a-pod-that-uses-pod-affinity